solidity/interfaces/IOracleAggregator.sol

```diff
@@ -56,6 +56,15 @@ interface IOracleAggregator is ITokenPriceOracle {
    */
   function availableOracles() external view returns (ITokenPriceOracle[] memory);
 
+  /**
+   * @notice Returns the oracle that would be assigned to the pair if `addOrModifySupportForPair`
+   *         was called by the same caller
+   * @param tokenA One of the pair's tokens
+   * @param tokenB The other of the pair's tokens
+   * @return The oracle that would be assigned (or the zero address if none could be assigned)
+   */
+  function previewAddOrModifySupportForPair(address tokenA, address tokenB) external view returns (ITokenPriceOracle);
```

Comment:
I'd add @dev about the order of tokenA and tokenB comment :).

Reply:
Done!